torch/_inductor/compile_fx.py

```diff
@@ -353,9 +367,25 @@ def compile_fx_inner(
         # doesnt work for non-trees because the warmup run would apply mutation twice
         if config.triton.cudagraph_trees:
             # checking if mutation is only on parameters/static inputs
-            has_mutation = not all(
-                idx < num_fixed for idx in compiled_graph.mutated_input_idxs
-            )
+            mutation_indices = [
+                idx for idx in compiled_graph.mutated_input_idxs if idx >= num_fixed
+            ]
+            has_mutation = len(mutation_indices) != 0
+            if has_mutation:
+                stack_trace: Optional[str] = ""
+                placeholders = [
+                    node for node in gm.graph.nodes if node.op == "placeholder"
+                ]
+
+                for idx in mutation_indices:
+                    placeholder = placeholders[idx]
+                    if stack_trace := get_mutating_use_stack_trace(placeholder):
+                        break
+
+                if stack_trace:
+                    msg = f"skipping cudagraphs due to mutaton on input. Found from : \n {stack_trace}"
+                    compiled_graph.disabled_cudagraphs_reason = msg
```

Comment:
Just to confirm: this PR does more than just "also log stack trace", right? Before this PR, it looks like since we weren't setting `compiled_graph.disabled_cudagraphs_reason` for input mutations, so we would lose the error message.

Reply:
Before the pr, we would just "skip because input mutation" but we skip.